components/mail/search-bar.tsx

```
@@ -18,7 +18,7 @@ import { useState } from "react";
 
 const inboxes = ["All Mail", "Inbox", "Drafts", "Sent", "Junk", "Trash", "Archive"];
 
-function DateFilter() {
+function DateFilter({ id }: { id: string }) {
```

ESLint

[error] 'id' is defined but never used.

(@typescript-eslint/no-unused-vars)